mods/default/nodes.lua

```diff
@@ -526,9 +526,9 @@ minetest.register_node("default:snow", {
 	},
 	groups = {crumbly = 3, falling_node = 1, puts_out_fire = 1, snowy = 1},
 	sounds = default.node_sound_dirt_defaults({
-		footstep = {name = "default_snow_footstep", gain = 0.15},
-		dug = {name = "default_snow_footstep", gain = 0.2},
-		dig = {name = "default_snow_footstep", gain = 0.2}
+		footstep = {name = "default_snow_footstep", gain = 0.2},
```

Comment:
This deserves a function: `default.node_sound_snow_defaults`

Reply:
I agree, will do.

Reply:
Done.